spec/requests/api/groups_spec.rb

```diff
@@ -0,0 +1,264 @@
+#
+# Rest API Request Tests - Groups specs
+#
+# - Creating a group                      /api/groups                           POST
+# - Creating a group via action           /api/groups                           action "create"
+# - Creating multiple groups              /api/groups                           action "create"
+# - Edit a group                          /api/groups/:id                       action "edit"
+# - Edit multiple groups                  /api/groups                           action "edit"
+# - Delete a group                        /api/groups/:id                       DELETE
+# - Delete a group by action              /api/groups/:id                       action "delete"
+# - Delete multiple groups                /api/groups                           action "delete"
+#
+require 'spec_helper'
+
+describe ApiController do
+  include Rack::Test::Methods
+
+  let(:expected_attributes) { %w(id guid description group_type tenant_id) }
+
+  let(:sample_group1) { {:description => "sample_group_1"} }
+  let(:sample_group2) { {:description => "sample_group_2"} }
+  let(:group1) { FactoryGirl.create(:miq_group, sample_group1) }
+  let(:group2) { FactoryGirl.create(:miq_group, sample_group2) }
+
+  let(:role3)    { FactoryGirl.create(:miq_user_role) }
+  let(:tenant3)  { FactoryGirl.create(:tenant, :name => "Tenant2") }
```

Comment:
Bit confused about this. `tenant3`'s name is `"Tenant2"`? Are there roles/tenants 1-2?